### DISCUSSION TITLE
I can't reverify my student identity

### DISCUSSION THREAD

**Discussion by Pan-x3** [community/community#53261] [GitHub Education]
I had received the "Academic Reverification Required"email,but when I try to reverify my student identity, I receive a warning (We require applicants of ... to use one of these school - issued email addresses to apply，Please select a different email address ).However, I don't have a school-issued email,so I can't reverify my identity

**Comment by Sazid99246** [comment]
Same issue 😥😥

**Comment by joshi-ji** [comment]
How did you uploaded your documents? There is not option to upload documents on the site.

**Comment by cobrat** [comment]
Same issue. Unable to find the "submit your information again" button on the webpage https://education.github.com/benefits.
